README.md

```diff
@@ -39,6 +39,7 @@ To test changes without install this dependency on your application, on your ter
   export AWS_ACCESS_KEY_ID=foo
   export AWS_SECRET_ACCESS_KEY=bar
   export AWS_ENDPOINT="http://localhost:4566"
+  export ENABLE_AWS_CLIENT_CONFIG=true
```

Comment:
The value of this environment variable is boolean here and in the rest of the code is a string, is a expected?

Reply:
That makes no difference, in the end, the value will be handled in a proper way.
Do you think that we must define this value as a String?

Reply:
No, I trust you